posix.mak

```diff
@@ -109,7 +109,7 @@ test_tests_extractor: $(ROOT)/tests_extractor
 
 test_rdmd: $(ROOT)/rdmd_test $(ROOT)/rdmd
 	$< --compiler=$(abspath $(DMD)) -m$(MODEL)
-	$(DMD) $(DFLAGS) -unittest -main -run rdmd.d
+	$(DMD) $(DFLAGS) -unittest -main -run rdmd.d rdmd_pre_dash_i.d
```

Comment:
This won't work, everything after first `-run` argument is treated as the program command line.